api/files/views.py

```diff
@@ -44,6 +44,9 @@ def get_file(self, check_permissions=True):
             if not isinstance(obj, BaseFileNode):
                 raise NotFound
 
+        if obj.node.is_quickfiles and not obj.node.creator.is_active:
```

Comment:
Should this check `obj.node.creator.is_disabled`? There are many reasons a user may be inactive, but we should only return a 410 if they are disabled.

Reply:
I just wanted to be as broad as possible in case we had some edge with accounts being unclaimed getting quickfiles or something, but given the specific nature of the error message you're right.